src/OpenToolkit.Mathematics/Geometry/Box3.cs

```diff
@@ -0,0 +1,234 @@
+//
+// Box3.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+using System.Globalization;
+using System.Runtime.InteropServices;
+
+namespace OpenToolkit.Mathematics
+{
+    /// <summary>
+    /// Defines a 3d box.
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct Box3 : IEquatable<Box3>
+    {
+        /// <summary>
+        /// The left boundary of the structure.
+        /// </summary>
+        public float Left;
+
+        /// <summary>
+        /// The right boundary of the structure
+        /// </summary>
+        public float Right;
+
+        /// <summary>
+        /// The top boundary of the structure
+        /// </summary>
+        public float Top;
+
+        /// <summary>
+        /// The bottom boundary of the structure
+        /// </summary>
+        public float Bottom;
+
+        /// <summary>
+        /// The back boundary of the structure
+        /// </summary>
+        public float Back;
+
+        /// <summary>
+        /// The front boundary of the structure
+        /// </summary>
+        public float Front;
```

Comment:
with these we only have AABB, so would min/max Vector3 not a lot easier to use and access?

Reply:
Will be fixed in #920